### PR TITLE
Added new Fastly CLI commands.

### PR DESCRIPTION
**Pull request by markdorison**
* datacenters
* ip-list
* edge-check

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "co": "^4.6.0",
     "co-prompt": "^1.0.0",
     "commander": "^2.9.0",
-    "fastly": "git+https://github.com/ChromaticHQ/fastly.git#85086829958a05faa53b445501d782e4772f356a",
+    "fastly": "git+https://github.com/ChromaticHQ/fastly.git#downstream-changes",
```
Comment (guschilds) [inline package.json:18]: I see what you chose there.

**Review by guschilds — APPROVED** [review]
No need to change `Fastly.prototype.request` after all? https://github.com/thisandagain/fastly/compare/develop...ChromaticHQ:additional-methods?expand=1

File: package.json (hunk 1 above)
Comment (markdorison) [reply package.json:18]: @guschilds For now.

**Comment by markdorison** [comment]
> No need to change Fastly.prototype.request after all?

@guschilds I chose not to include the method I was troubleshooting with you at this time.

**Pull request merged**